pyproject.toml

```diff
@@ -60,6 +60,7 @@ pydantic = "<=1.8.2"
 analytics-python = "^1.4.0"
 distro = "^1.6.0"
 tabulate = "^0.8.9"
+rich = "^11.0.0"
```

Comment:
Do we already want this in even before we're using it? :-) I would keep it out for now

Reply:
I'm actually using it in the `drift_detection` example.